misc/packaging/100_ACRN

```diff
@@ -27,8 +27,8 @@ while IFS= read -r line; do
     fi
 done < <(blkid |grep ext4 |grep ${type})
 
-if ls /boot/vmlinuz*acrn-sos* 1> /dev/null 2>&1;then
-    sos_kernel=$(ls -tr /boot/vmlinuz-*acrn-sos* | tail -1)
```

Comment:
We should update `sos_kernel` to `service_vm_kernel` to improve readibility and be consistent.

Reply:
Thanks,  It has been fixed.